docs/en/latest/plugins/error-log-logger.md

```diff
@@ -30,31 +30,38 @@ title: error-log-logger
-| keepalive        | integer | optional    | 30      | [1,...] | Time for keeping the cosocket alive, unit: second.                                                   |
-| level            | string  | optional    | WARN    |         | The filter's log level, default warn, choose the level in ["STDERR", "EMERG", "ALERT", "CRIT", "ERR", "ERROR", "WARN", "NOTICE", "INFO", "DEBUG"], the value ERR equals ERROR.         |
-| tls              | boolean | optional    | false   |         | Control whether to perform SSL verification                                              |
-| tls_server_name  | string  | optional    |         |         | The server name for the new TLS extension  SNI                                                            |
-| batch_max_size   | integer | optional    | 1000    | [1,...] | Max size of each batch                                                                   |
-| inactive_timeout | integer | optional    | 3       | [1,...] | Maximum age in seconds when the buffer will be flushed if inactive                       |
-| buffer_duration  | integer | optional    | 60      | [1,...] | Maximum age in seconds of the oldest entry in a batch before the batch must be processed |
-| max_retry_count  | integer | optional    | 0       | [0,...] | Maximum number of retries before removing from the processing pipe line                  |
-| retry_delay      | integer | optional    | 1       | [0,...] | Number of seconds the process execution should be delayed if the execution fails         |
+| Name                             | Type    | Requirement | Default                        | Valid   | Description                                                                                          |
+| -------------------------------- | ------- | ----------- | ------------------------------ | ------- | ---------------------------------------------------------------------------------------------------- |
+| tcp.host                         | string  | required    |                                |         | IP address or the Hostname of the TCP server.                                                        |
+| tcp.port                         | integer | required    |                                | [0,...] | Target upstream port.                                                                                |
+| tcp.tls                          | boolean | optional    | false                          |         | Control whether to perform SSL verification.                                                         |
+| tcp.tls_server_name              | string  | optional    |                                |         | The server name for the new TLS extension  SNI.                                                      |
+| skywalking.endpoint_addr         | string  | required    | http://127.0.0.1:12900/v3/logs |         | the http endpoint of Skywalking.                                                                     |
```

Comment:
Once you provide a default value it is no longer required.

Reply:
Got it.